mantidproject/reports
# Add application to feature usage

This adds an application column to the feature usage table and updates the views to copr with http requests which contain this information.

This also updates the server to run in python3.

To test:

- Follow the instructions in the readme to setup a local server build of master.
- Populate the server database with some entries by changeing the report url in mantid and using some features.
- Switch to this branch and tear down and re-up the server (use bin/shutdown.sh and bin/boot.sh)
- Check that the database has been updated to add an extra column to the features table for application and that all existing entries are set to mantidplot
- With an old version of mantid try sending some new feature requests check that they are still processed correctly by the server and that their application defaults to mantidplot
- Update mantid to this branch https://github.com/mantidproject/mantid/pull/27308
- Try sending some new requests from mantidplot, mantidworkbench and python scripts. Check that they arrive with the correct application.
- Check that the other functionality of the server still works.
